src/client.rs

```diff
@@ -607,6 +617,8 @@ pub struct ConnectOptions {
     user: Option<(String, String)>,
     #[cfg(feature = "tls")]
     tls: Option<TlsOptions>,
+    /// (keep_alive_timeout, keep_alive_interval)
```

Comment:
According to the code, the comment should be `(keep_alive_interval, keep_alive_timeout)`.